test/skimNtuple2016_HHbtag.cpp

```diff
@@ -1465,15 +1381,15 @@ int main (int argc, char** argv)
       int dauType = theBigTree.particleType->at(idau);
       if (oph.isMuon(dauType))
       {
-	bool passMu   = oph.muBaseline (&theBigTree, idau, 19., 2.3, 0.15, OfflineProducerHelper::MuTight, string("All") , (DEBUG ? true : false));
+	bool passMu   = oph.muBaseline (&theBigTree, idau, 20., 2.4, 0.15, OfflineProducerHelper::MuTight, string("All") , (DEBUG ? true : false));
 	bool passMu10 = oph.muBaseline (&theBigTree, idau, 10., 2.4, 0.30, OfflineProducerHelper::MuTight, string("All") , (DEBUG ? true : false));
 
 	if (passMu) ++nmu;
 	else if (passMu10) ++nmu10;
       }
       else if (oph.isElectron(dauType))
       {
-	bool passEle   = oph.eleBaseline (&theBigTree, idau, 25., 2.3, 0.1, OfflineProducerHelper::EMVATight, string("Vertex-LepID-pTMin-etaMax") , (DEBUG ? true : false));
+	bool passEle   = oph.eleBaseline (&theBigTree, idau, 20., 2.5, 0.1, OfflineProducerHelper::EMVATight, string("Vertex-LepID-pTMin-etaMax") , (DEBUG ? true : false));
```

Comment:
We need to verify that trigger and id+iso scale factors exist up to eta 2.5

Reply:
[Twiki](https://twiki.cern.ch/twiki/bin/viewauth/CMS/EgHLTScaleFactorMeasurements) for electrons

- [x] UL trigger SFs: From the files I inspected it seems we always have (Legacy) the "EtaGt2p1" label (including 2016), so no error should be thrown. The new UL legacy files (@portalesHEP suggested using the ones from Ka Wa Ho) go up to 2.5.

- [x] UL id/iso SFs: the versions we currently have are already Ultra-Legacy. Both ID and ISO SFs for electrons (merged into single file) go up to 2.5, so all good!

Reply:
Thanks also for checking here. The error was probably coming from muons then. In any case we should move to the UL SF ASAP.